system-x/services/redis/src/main/java/software/tnb/redis/validation/RedisValidation.java

```diff
@@ -0,0 +1,37 @@
+package software.tnb.redis.validation;
+
+import software.tnb.common.validation.Validation;
+
+import io.lettuce.core.RedisClient;
+import io.lettuce.core.api.StatefulRedisConnection;
+
+public class RedisValidation implements Validation {
+
+    private RedisClient redisClient;
+    private StatefulRedisConnection<String, String> connection;
+    private String host;
+    private int port;
+
+    public RedisValidation(RedisClient redisClient, String host, int port) {
+        this.redisClient = redisClient;
+        this.host = host;
+        this.port = port;
+        this.connection = redisClient.connect();
```

Comment:
same as in ldap, connect in `openResources` method

Reply:
Done